src/components/Playground.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react'
+import AceEditor from 'react-ace'
+
+import 'brace/mode/javascript'
+import 'brace/theme/github'
+
+/**
+ * A prop that is passed to the Playground
+ * @property editorValue - The string content of the react-ace editor
+ * @property updateCode  - A callback function for the react-ace editor's `onChange`
+ */
+export interface IPlaygroundProps {
+  editorValue: string
+  updateCode: (newCode: string) => void
+}
+
+/**
+ * A component representing the Playground
+ */
+export default class Playground extends React.Component<IPlaygroundProps, {}> {
```

Comment:
You should always default to `React.SFC` until you need state

Reply:
In this case, our editor would need to call upon a `dispatch` to update the `editorValue` within the `IPlaygroundState`, whenever someone edits code in the ace editor. Is a Component warranted in this case?

Reply:
Yeah, so in this case the editor component is stateful, but the playground component itself should be stateless

Reply:
Ah, I understand now. Will do!